src/content/governance/index.md

```diff
@@ -0,0 +1,151 @@
+
+As the champion of a contentious EIP, you should try and build consensus, address objections and document community support for your proposal. Given the size and diversity of the Ethereum community, there isn't a single metric (e.g. a coin vote) that can be used, and EIP Champions are expected to adapt to the circumstances of their proposal.
+
+Beyond the security of the Ethereum network, which contentious forks may compromise, significant weight is usually placed on what Application/Tooling Developers and Application Users value, given that their using and developing on Ethereum is what makes the ecosystem attractive for other stakeholders.
+
+<Divider />
+
+## Handling disagreements {#disagreements}
+
+Having many stakeholders with different motivations and beliefs means that disagreements are not uncommon.
+
+Generally, disagreements are handled with long-form discussion to understand the root of the problem. Typically, one group concedes, or a happy medium is achieved. If one group feels strongly enough, implementing a particular change on Ethereum could result in a split chain.
+
+### The DAO fork {#dao-fork}
+
+The DAO fork was in response to the [2016 DAO attack](https://www.coindesk.com/understanding-dao-hack-journalists) where an insecure [DAO](/glossary/#dao) contract was drained of over 3.6 million ETH in a hack. The fork moved the funds from the faulty contract to a new contract allowing anyone who lost funds in the hack to recover them.
+
+This course of action was voted on by the Ethereum community. Any ETH holder was able to vote via a transaction on [a voting platform](http://v1.carbonvote.com/). The decision to fork reached over 85% of the votes.
+
+Some miners refused to fork because the DAO incident wasn't a defect in the protocol. They went on to form [Ethereum Classic](https://ethereumclassic.org/).
+
+Watch more on the DAO hack:
+
+<iframe width="100%" height="315px" src="https://www.youtube.com/embed/rNeLuBOVe8A" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<Divider />
+
+## Beacon Chain Development {#beacon-chain}
+
+The Ethereum governance process often trades off speed and efficiency for openness and inclusivity. In order to accelerate the development of the Beacon Chain, it was launched separately from the proof of work Ethereum network and followed its own governance practices.
+
+While the specification and implementations development has always been fully open source, the formal processes used to propose updates described above weren't used. This allowed changes to be specified and agreed upon quicker by researchers and implementers.
+
+When the Beacon Chain merges with the Ethereum execution layer, the governance process to propose changes will be harmonized.
```

Comment:
Perhaps we should say this is already happening, e.g. how the merge as already been created as an EIP: https://github.com/ethereum/EIPs/pull/3675

Reply:
Agree. This info is also missing from the actual merge page.

Reply:
Another thing you can add here is that we are creating beacon chain-like specs for the execution layer too, see: https://ethereum.github.io/execution-specs/

Reply:
does this mean that the EIP process will be deprecated?

Reply:
Not necessarily, but it could look something like consensus changes also needing an EIP for a description (not currently the case) and/or execution changes needing a PR to the specs with the technical details (again, not currently the case). That's my favoured approach, but obviously it will need to be discussed further.